### ISSUE TITLE
Material design

### ISSUE THREAD

**Issue by Ejsstiil** [Snoothy/UCR#147]
First of all i want to thank you for such great program!

My question is addressed to "material" design. Can you add an option into the settings or commandline option for use a native win skin (ie. no skin at all) Just native?
Because in my opinion:
- its a huge waste of space on the screen
- some mid-sized project inside requires to endlessly scroll up and down which also adds complexity when you see only a fraction of plugins (hardly two)
- for me it looks inappropriate on windows (we are at Windows not on Android)
- personally hate any forced antialiasing of anything on the desktop

Others:
- windows should retain its positions (they even always opens on second display with dual-monitor setup). They should follow same display as the main app is opened from
- i would prefer more compact look, less or none padding, smaller font, antialiasing on/off, dark theme in case you decline get rid-off this design
- do you plan to allow users to create skins?

**Comment by hiilikewiki** [comment]
Haven't really looked at the source code but from what I have seen while using it could be Winforms in which case you're in luck. 
When you mention creating skins, as with a config file with the colours, measurements etc or for example an ISkin which would be used by assemblies which UCR would load and add to the available list of skins?

**Comment by StrawberryCough** [comment]
I'd seriously cry tears of joy if I could do 3 simple things with the UI:

- Dark Theme (My eyes!!!)
- Hotkeys to globally scale the UI
- Collapsible Mappings List in Edit Profile

**Comment by 6wheels** [comment]
I also would to be able to collapse mapping, with a top button to collapse/expand all and a button on each individual mapping. Would be great.

I agree that for complex profiles, we have to scroll a lot, which is not really user friendly.

Otherwise, this app is great!

**Comment by phasermaniac83** [comment]
Yes, I agree, everything it's too big. Collapse button would help too.

**Comment by Ejsstiil** [comment]
Too bad - project looks dead unfortunately
